content/en/user-guide/aws/stepfunctions/index.md

```diff
@@ -130,6 +130,7 @@ LocalStack's Step Functions emulation supports the following AWS services:
 |                                | Amazon EventBridge      | &#10003;         |                   |                    | &#10003;                              |
 |                                | AWS Glue                | &#10003;         | &#10003;          |                    |                                       |
 |                                | AWS Step Functions      | &#10003;         | &#10003;          | &#10003;           | &#10003;                              |
+|          | AWS Batch            | &#10003;         | &#10003;          |                    |       &#10003;                                |
```

Comment:
Thank you for adding this in the docs! 
Like AWS, we don't support `.waitForTaskToken` for AWS Batch, just `Request Response` and `.sync`